os/net/lwip/configs/socket/Kconfig

```diff
@@ -11,11 +11,11 @@ config NET_SOCKET
 
 if NET_SOCKET
 
-config NSOCKET_DESCRIPTORS
-	int "Number of socket descriptor"
+config NLWIP_DESCRIPTORS
```

Comment:
If NSOCKET_DESCRIPTOR was removed then the prefix of CONFIG_NSOCKET_DESCRIPTOR should be removed too. the prefix CONFIG_ should be used to Kconfig definition only.

Reply:
CONFIG_NSOCKET_DESCRIPTOR will be defined at `config.h` by `mkconfig`.
I think the CONFIG prefix should be added because the definition of NSOCKET_DESCRIPTOR only exists at `config.h`.
For example, CONFIG_SMARTFS_NLOGGING_SECTORS isn't Kconfig definition, but it has the CONFIG prefix.